### PR TITLE
WT-6747 Add prepare support in checkpoint test

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: test/checkpoint/test_checkpoint.c [hunk 1]
```diff
@@ -333,7 +336,7 @@ static int
 usage(void)
 {
     fprintf(stderr,
-      "usage: %s [-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys]\n\t[-l log] [-n ops] "
+      "usage: %s [-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys]\n\t[-l log] [-n ops]"
```
Comment (tetsuo-cpp) [inline test/checkpoint/test_checkpoint.c:339]: Nit: I think you still want the space here to separate the next arg.

File: test/checkpoint/test_checkpoint.c [hunk 2]
```diff
@@ -333,7 +336,7 @@ static int
...
       "[-r runs] [-T table-config] [-t f|r|v]\n\t[-W workers]\n",
```
Comment (tetsuo-cpp) [inline test/checkpoint/test_checkpoint.c:340]: Should probably update this part with `|p`.

File: test/checkpoint/workers.c [hunk 3]
```diff
@@ -263,11 +263,23 @@ real_worker(void)
             goto err;
         } else if (ret == 0) {
             next_rnd = __wt_random(&rnd);
-            if (next_rnd % 7 != 0) {
+            if (next_rnd % 7 == 0) {
```
Comment (tetsuo-cpp) [inline test/checkpoint/workers.c:266]: What's the reasoning behind this?
Looks like we'll do timestamped updates less frequently with this change.

**Review by tetsuo-cpp — COMMENTED** [review]
Seems reasonable.
Just had a few minor comments.

File: test/checkpoint/test_checkpoint.c (hunk 2 above)
Comment (quchenhao) [reply test/checkpoint/test_checkpoint.c:340]: I was planning to do that but I think we only options with parameters here. `f|r|v` are table types not options.

File: test/checkpoint/workers.c (hunk 3 above)
Comment (quchenhao) [reply test/checkpoint/workers.c:266]: No, we will commit less frequently with this change. I'm not sure why before we commit in 6/7 chance. I want each commit to have more operations to trigger the bug we see in WT-6733.

File: test/checkpoint/test_checkpoint.c (hunk 2 above)
Comment (tetsuo-cpp) [reply test/checkpoint/test_checkpoint.c]: Ah, I see.

File: test/checkpoint/workers.c (hunk 3 above)
Comment (tetsuo-cpp) [reply test/checkpoint/workers.c:266]: Yeah, that makes sense.
I lost track of the braces.

**Review by tetsuo-cpp — APPROVED** [review]
LGTM.

**Pull request merged**